README.md

```diff
@@ -19,3 +19,102 @@ problems.  Some of the things it does are:
   - Ability to request tasks of the bot, such as setting of labels, assignment,
     and milestone, which would otherwise not be possible without commit rights
     to the repo.
+
+## Development
+
+### Prerequisites
+
+* Membership in the miq-test organization
+* ruby 1.9.3 with bundler
+* redis ~> 2.8.10
+* postgresql
```

Comment:
shall i remove the miq-test org requirement, and make it more generic, i.e. something like you need _a_ sandbox repo to test it out?